### PR TITLE
Update postman collection url

### PR DESCRIPTION
**Pull request by Ciaran-Mooney**
Update the collection url since it has been updated and the previous link no longer works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Verify Account Balance, Account Owner and Account Income through our simple API.
...
+1. Click here: [<img src=https://run.pstmn.io/button.svg alt="Run In Postman" style="width: 128px; height: 32px;">](https://god.gw.postman.com/run-collection/26779181-1d6e0d10-6801-491e-a507-eae59c689c81?action=collection%2Ffork&source=rip_markdown&collection-url=entityId%3D26779181-1d6e0d10-6801-491e-a507-eae59c689c81%26entityType%3Dcollection%26workspaceId%3Ddc9c22f1-d88f-417d-8a88-662dba528689#?env%5BMastercard%20Open%20Banking%20EU%20Environment%20-%20Demo%5D=W3sia2V5IjoiYmFzZV91cmwiLCJ2YWx1ZSI6Imh0dHBzOi8vYXBpLXNhbmRib3guYWlpYS5ldSIsImVuYWJsZWQiOnRydWUsInR5cGUiOiJ0ZXh0Iiwic2Vzc2lvblZhbHVlIjoiaHR0cHM6Ly9hcGktc2FuZGJveC5haWlhLmV1Iiwic2Vzc2lvbkluZGV4IjowfSx7ImtleSI6ImNsaWVudF9pZCIsInZhbHVlIjoiIiwiZW5hYmxlZCI6dHJ1ZSwidHlwZSI6InRleHQiLCJzZXNzaW9uVmFsdWUiOiIiLCJzZXNzaW9uSW5kZXgiOjF9LHsia2V5IjoiY2xpZW50X3NlY3JldCIsInZhbHVlIjoiIiwiZW5hYmxlZCI6dHJ1ZSwidHlwZSI6InRleHQiLCJzZXNzaW9uVmFsdWUiOiIiLCJzZXNzaW9uSW5kZXgiOjJ9LHsia2V5IjoiY2xpZW50X3JlZGlyZWN0X3VyaSIsInZhbHVlIjoiIiwiZW5hYmxlZCI6dHJ1ZSwidHlwZSI6InRleHQiLCJzZXNzaW9uVmFsdWUiOiIiLCJzZXNzaW9uSW5kZXgiOjN9XQ==)
```
Comment (krlaiia) [inline README.md:66]: It points to "Mastercard Open Banking EU - Demo" not to "Mastercard Open Banking EU Enterprise". It that ok?

Comment (Ciaran-Mooney) [reply README.md:66]: Yes that's ok - I just want to keep the original collection that was linked working.

Comment (flkdrn87) [reply README.md:66]: Can we point to "Mastercard Open Banking EU Enterprise" ?

Comment (flkdrn87) [reply README.md:66]: Looks good to me

**Review by fileme — APPROVED** [review]
Approved.

**Review by fileme — APPROVED** [review]
Approved

**Review by karen-avetisyan-mc — APPROVED** [review]

**Pull request merged**